akka-actor/src/main/scala/akka/serialization/Serializer.scala

```diff
@@ -351,7 +351,9 @@ final case class DisabledJavaSerializer(system: ExtendedActorSystem) extends Ser
   }
 
   override def fromBinary(buf: ByteBuffer, manifest: String): AnyRef = {
```

Comment:
@ironfish could you remove the `clazz` logging 4 lines above also.
remove `+ "Message class was: [{}]", clazz`